src/scripts/main.js

```diff
@@ -1,3 +1,36 @@
 'use strict';
 
-// write your code here
+const logo = document.querySelector('.logo');
+const body = document.querySelector('body');
+
+const promise1 = new Promise((resolve) => {
+  logo.addEventListener('click', () => {
+    resolve('Promise was resolved!');
+  });
+});
+
+const promise2 = new Promise((resolve, reject) => {
+  setTimeout(() => {
+    reject(new Error('Promise was rejected!'));
+  }, 3000);
+});
+
+const handleSuccess = () => {
+  const message = document.createElement('div');
+
+  message.className = 'message';
+  message.textContent = 'Promise was resolved!';
+  body.appendChild(message);
+};
+
+const handleError = () => {
+  const message = document.createElement('div');
+
+  message.className = 'message error-message';
+  message.textContent = 'Promise was rejected!';
+  body.appendChild(message);
+};
+
+promise1.then(handleSuccess).catch(handleError);
```

Comment:
The catch block for promise1 will never execute because the promise only resolves. Consider removing the catch block or modifying the promise to potentially reject.